### PR TITLE
Check for existing decompressed acquire when decrypting

### PR DESCRIPTION
**Pull request by Poeloe**

### CODE REVIEW

File: acquire/tools/decrypter.py [hunk 1]
```diff
@@ -244,10 +245,22 @@ def decrypt_header(header, fingerprint, key_file=None, key_server=None):
         return base64.b64decode(result["header"])
 
 
+def check_existing(in_path: Path, out_path: Path, status_queue: Queue) -> bool:
+    if out_path.exists():
+        _info(status_queue, f"Output file already exists: {out_path}")
+        return True
+
+    # Check if acquire file is compressed. If so, check if decompressed file already exists
+    if in_path.stem.endswith((".tgz", ".gz")) and (decompressed_file := out_path.with_suffix("")).exists():
```
Comment (Schamper) [inline acquire/tools/decrypter.py:254]:
```suggestion
    if in_path.suffix in (".tgz", ".gz") and (decompressed_file := out_path.with_suffix("")).exists():
```

**Review by Schamper — CHANGES_REQUESTED** [review]

File: acquire/tools/decrypter.py (hunk 1 above)
Comment (Schamper) [reply acquire/tools/decrypter.py:254]: I've been corrected offline, but can you add a comment with an example? E.g. 
`Path("file.tar.gz.enc").stem -> "file.tar.gz"`

Comment (Poeloe) [reply acquire/tools/decrypter.py:255]: Added this to the existing comment 😄

File: acquire/tools/decrypter.py [hunk 2]
```diff
@@ -250,7 +250,13 @@ def check_existing(in_path: Path, out_path: Path, status_queue: Queue) -> bool:
         _info(status_queue, f"Output file already exists: {out_path}")
         return True
 
-    # Check if acquire file is compressed (Path("file.tar.gz.enc").stem -> "file.tar.gz") 
+    # If suffixes of the out_path do not correspond with the in_path suffixes (minus ".enc"),
+    # we're probably dealing with a special custom filename. Therefore, do not check for the existence of the
```
Comment (Horofic) [inline acquire/tools/decrypter.py:254]: Could you add this case as a test as well.

**Review by Horofic — CHANGES_REQUESTED** [review]

File: acquire/tools/decrypter.py [hunk 3]
```diff
@@ -244,10 +245,29 @@ def decrypt_header(header, fingerprint, key_file=None, key_server=None):
         return base64.b64decode(result["header"])
 
 
+def check_existing(in_path: Path, out_path: Path, status_queue: Queue) -> bool:
```
Comment (Schamper) [inline acquire/tools/decrypter.py:248]:
```suggestion
def check_existing(in_path: Path, out_path: Path, status_queue: multiprocessing.Queue) -> bool:
```

File: acquire/tools/decrypter.py [hunk 4]
```diff
@@ -12,6 +12,7 @@
 from collections import defaultdict, deque
 from concurrent.futures import ProcessPoolExecutor
 from datetime import datetime, timezone
+from multiprocessing import Queue
```
Comment (Schamper) [inline acquire/tools/decrypter.py:15]:
```suggestion
```

**Review by Schamper — CHANGES_REQUESTED** [review]

**Review by Schamper — APPROVED** [review]

**Review by Horofic — APPROVED** [review]

**Pull request merged**